Mathlib/Topology/ContinuousFunction/Basic.lean

```diff
@@ -469,6 +469,49 @@ end Gluing
 
 end ContinuousMap
 
+namespace QuotientMap
+
+variable {X Y Z : Type*} [TopologicalSpace X] [TopologicalSpace Y] [TopologicalSpace Z]
+    {f : C(X, Y)} (hf : QuotientMap f) (g : C(X, Z)) (h : ∀ a b, f a = f b → g a = g b)
+
+/-- The homeomorphism from the quotient of a quotient map to its codomain. -/
+noncomputable
+def homeomorph : Quotient (Setoid.ker f) ≃ₜ Y where
+  toEquiv := Setoid.quotientKerEquivOfSurjective _ hf.surjective
+  continuous_toFun := quotientMap_quot_mk.continuous_iff.mpr hf.continuous
+  continuous_invFun := by
+    rw [hf.continuous_iff]
+    convert continuous_quotient_mk'
+    ext
+    simp only [Equiv.invFun_as_coe, Function.comp_apply,
+      (Setoid.quotientKerEquivOfSurjective f hf.surjective).symm_apply_eq]
+    rfl
+
+/-- Descend a continuous map, which is constant on the fibres, along a quotient map. -/
+noncomputable
+def descend : C(Y, Z) where
```

Comment:
I think elsewhere in mathlib, this is called `lift`! See for instance [`RingHom.liftOfSurjective`](https://leanprover-community.github.io/mathlib4_docs/Mathlib/RingTheory/Ideal/Operations.html#RingHom.liftOfSurjective), which also states it as an equivalence.